Item23_Understand_std_move_and_std_forward/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-set_source_files_properties(std_move_in_cpp14.cpp PROPERTIES COMPILE_FLAGS "-std=c++14 -stdlib=libc++")
+set_source_files_properties(std_move_in_cpp14.cpp PROPERTIES COMPILE_FLAGS "-std=c++14")
```

Comment:
Dunno, but if u want build with GCC (4.9), you have to remove this option (only available on clang compiler (>= 3.5)).
Don't forget, my pull request come from a specific branch 'gcc-4.9' so what it's present on this branch, this is only ("necessary") for gcc-4.9 compiler.
I think a good strategy can be to divided your depot with sub-branches. Each sub-branch will be dedicated for a specific compiler (and version, maybe platform, etc ...).
After this step, the next step can be to merge some sub-branches to get a 'global' compatibility.